.github/workflows/test-tpg.yml

```
@@ -0,0 +1,47 @@
+name: Build and Unit Test GA Provider
```

go/github-docs/security-recommendations#toolssettings-9

```suggestion
name: Build and Unit Test GA Provider

permissions: read-all
```